addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -67,7 +67,10 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false)
         if (isUnremovable(leftNode)) {
             throw UNREMOVABLE_ROLLBACK_CODE;
         }
-        if (isMediaElement(leftNode)) {
+        if (
+            isMediaElement(leftNode) ||
+            (leftNode.getAttribute && leftNode.getAttribute('contenteditable') === 'false')
```

Comment:
Wouldn't this fail for <my-node contenteditable="0"> and <my-node contenteditable="False"> for instance? Aren't those valid forms?

Reply:
for `<my-node contenteditable="0">` is equivalent to `<my-node>` because the attribute is not valid.
`<my-node contenteditable="False">` is indeed equivalent to `<my-node contenteditable="false">`. I have changed the code accordingly.